arrow/obsidianmodule.c

```diff
@@ -229,11 +229,17 @@ Obsidian_evolve(ObsidianObject *self, PyObject *args)
   Py_XDECREF(state_array);
 
   // Construct the return value that will be ultimately visible to python
-  return Py_BuildValue("iOOO",
-                       result.steps,
-                       time_obj,
-                       events_obj,
-                       outcome_obj);
+  PyObject * ret = Py_BuildValue("iOOO",
```

Comment:
Nit: If we had a C/C++ style guide, it would write this as `PyObject *ret = ...`

Reply:
There I go multiplying pointers again. I'll conform it with the rest of the code (old habits die hard).

Reply:
> (Not saying I told you so. I would never do that.)

Throwing shade is the far preferred method. 

But then we wouldn't have learned all these wonderful things about the internals of the python interpreter! I consider this well earned. And yeah I'll do it in cython next time ; )